src/Auth0.ManagementApi/Models/AttackProtection/BreachedPasswordDetection.cs

```diff
@@ -28,6 +45,12 @@ public class BreachedPasswordDetection
         /// </summary>
         [JsonProperty("method")]
         public string Method { get; set; }
+
+        /// <summary>
+        /// Holds per-stage configuration options (shiels).
+        /// </summary>
+        [JsonProperty("stage")]
+        public Stage Stage { get; set; }
```

Comment:
Shouldn't this use `BreachedPasswordDetectionStage` ?

```suggestion
        public BreachedPasswordDetectionStage Stage { get; set; }
```

Reply:
Yeah I wasn't sure what was right here, I only used `BreachedPasswordDetectionStage` because of the clash with `Stage` in `SuspiciousIpThrottling.cs`, but I guess it makes sense to align the naming for both